aiida_vasp/parsers/parser_settings.py

```diff
@@ -33,7 +33,7 @@
         },
         'vasprun.xml': {
             'parser_class': VasprunParser,
-            'is_critical': True,
+            'is_critical': False,
```

Comment:
'is_critical' determines whether parsing will be aborted if this file does not exist. This is used as a measure for whether the VaspCalculation has run at all. Previously we have ben checking for both OUTCAR and vasprun.xml. Since the parsevasp update this has been lifted to only checking for vasprun.xml. We should check for at least one of those.

Reply:
Yes, good point. I do not think I have corrected this in the recent local version. Thanks.